utopia-remix/app/util/useSortCompareProject.tsx

```diff
@@ -0,0 +1,45 @@
+import moment from 'moment'
```

Comment:
Minor thing, but we use kebab-case in filenames not camel-case

Reply:
Ignore me, this is the remix app :)

Reply:
no it's good, I just realized filename casing is mixed 🤢 I'll use kebab for this one and have another cleanup PR